bussiness-logic-module/src/main/java/edu/hillel/App.java

```diff
@@ -38,5 +42,24 @@ public static void main( String[] args )
 //        // Виклик методу біна
 //        System.out.println(itemService.getAllItems());
 
+//        ItemRepository itemRepository = ItemRepositoryTxtFilesStorageImpl.getSingeltonInstance();
+//        ItemRepositoryTxtFilesStorageImpl itemRepositoryTxtFilesStorage = new ItemRepositoryTxtFilesStorageImpl();
+//        itemRepositoryTxtFilesStorage.getAllItems();
+//        ItemService itemService = ItemService.getSingletonInstance(itemRepository);
+//        itemService.getAllItems().forEach(item -> System.out.println(item));
+
+        UserRepository userRepository = UserRepositoryTxtFilesStorageImpl.getSingletonInstance();

```

Comment:
Again, write tests for implementet fiatures don`t use main method, at least not in PR

Reply:
done